DebugSwift/Sources/Helpers/Managers/FeatureHandling.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  FeatureHandling.swift
+//  DebugSwift
+//
+//  Created by Mochamad Rakha Luthfi Fahsya on 24/01/24.
+//
+
+import UIKit
+import CoreLocation
+
+final class FeatureHandling {
+    static let shared = FeatureHandling()
+    
+    func allFeatureHandler() {
+        UIView.swizzleMethods()
+        UIWindow.db_swizzleMethods()
+        URLSessionConfiguration.swizzleMethods()
+        CLLocationManager.swizzleMethods()
+        LogIntercepter.shared.start()
+        
+        NetworkHelper.shared.enable()
+        
+        CrashManager.register()
+    }
+    
+    func selectedFeatureHandler(viewController: String?) {
+        switch viewController {
+        case "network-title":
+            URLSessionConfiguration.swizzleMethods()
+            NetworkHelper.shared.enable()
+        case "":
+            break
+        default:
+            allFeatureHandler()
+        }
+    }
+    
+    func hideFeatureByIndex(indexArr: [Int]?) {
```

Comment:
Instead Int, you can use an Enum, like this:

public enum DebugSwiftFeatures: Int {
          case networking
          case performance
          case interface
          case resources
          case app
} 

This way it will be more specific which features that will be removed.

Reply:
and you can improve this function to be:

`hide(features: DebugSwiftFeatures...) {}`

In use:

FeatureHandling.hide(.app, .interface)

Reply:
Thank you for your advice, I will make the improvements as soon as possible 🚀.